ConsoleApp2/ConsoleApp2/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace ConsoleApp2
+{
+    
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine(mathfunction(mathfunction(mathfunction(mathfunction(5, 4), 3), 2), 0));
+        }
+
+        public static double mathfunction (double i, double y)
```

Comment:
В C# методы принято именовать с большой буквы, используя способ "camelCase".
```suggestion
        public static double MathFunction (double i, double y)
```